### PR TITLE
feat!: Update to Uno 5

### PR DESCRIPTION
**Pull request by juliecantin**
GitHub Issue: #

## Proposed Changes

Feature

## What is the current behavior?

Nventive.View targets Uno 4 and .net 6, supports Xamarin.

## What is the new behavior?

Nventive.View targets Uno 5 and .net 7, doesn't support Xamarin.

## Checklist

Please check if your PR fulfills the following requirements:

- [x] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [ ] ~~Contains **NO** breaking changes~~
- [x] Updated the Changelog
- [ ] Associated with an issue

Apps need to [migrate to Uno 5](https://platform.uno/docs/articles/migrating-to-uno-5.html).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,15 +4,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.0.0]
```
Comment (jeanplevesque) [inline CHANGELOG.md]: The build output isn't 1.0.0. It's currently still 0.5.0 and needs to be changed to 0.6.0.

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: src/View.Uno.WinUI/View.Uno.WinUI.csproj [hunk 2]
```diff
@@ -53,61 +54,45 @@
 		<DefineConstants>$(DefineConstants);WINDOWS_WINUI</DefineConstants>
 	</PropertyGroup>
 
-	<PropertyGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
```
Comment (jeanplevesque) [inline src/View.Uno.WinUI/View.Uno.WinUI.csproj:56]: Keep WASM.

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (carlh98) [reply CHANGELOG.md]: since it's a major change should it not be 1.0.0?

File: build/gitversion.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿assembly-versioning-scheme: MajorMinorPatch
 mode: ContinuousDeployment
-next-version: 0.5.0
+next-version: 1.0.0
```
Comment (jeanplevesque) [inline build/gitversion.yml]: This package is still in pre-release. The `next-version` should be 0.6.0.

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

**Review by CBrunet2212 — APPROVED** [review]

**Review by Soap-141 — APPROVED** [review]

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**